src/lib/pagination/ctrl.coffee

```diff
@@ -0,0 +1,99 @@
+EventsEmitter = require( 'events' )
+Pagination = require( './index' )
+PaginationView = require( './view' )
+
+###
+ * #PaginationCtrl
+ * Creates a pagination element and controls it.
+ *
+ * ## Events
+ *
+ * | CONSTANT.argument  | Description |
+ * |--------------------| -------------|
+ * | CHANGE | When page number is changed by user |
+ * | CHANGE.current  | New page number |
+ * | CHANGE.last | Number of last page |
+###
+class PaginationCtrl extends EventsEmitter
```

Comment:
Is this controller extending from EventsEmitter just for the @emit in the _onPageChange method?

Something looks wrong (for me) when I see a controller extending from a totally isolated class like EventsEmitter. Wouldn't be better to pass the EventsEmitter in the constructor or to create the instance there?